.github/workflows/wheels.yml

```diff
@@ -57,7 +57,7 @@ jobs:
       - uses: dtolnay/rust-toolchain@stable
       - name: Install cibuildwheel
         run: |
-          python -m pip install cibuildwheel==2.21.3
+          python -m pip install cibuildwheel==2.23.2
```

Comment:
This is from #1433, synch to HEAD from https://github.com/Qiskit/rustworkx.git not to include this